src/rules.jl

```diff
@@ -405,7 +405,7 @@ weight decay regularization.
                          (no need to change default)
 """
 ADAMW(η = 1f-3, β = (9f-1, 9.99f-1), γ = 0, ϵ = eps(typeof(η))) =
-  OptimiserChain(ADAM{typeof(η)}(η, β, ϵ), WeightDecay(γ))
+  OptimiserChain(ADAM{typeof(η)}(1, β, ϵ), WeightDecay{typeof(η)}(γ), Descent(η))
```

Comment:
```suggestion
  OptimiserChain(ADAM{typeof(η)}(1, β, ϵ), WeightDecay{typeof(η)}(γ), Descent(η))
```
Sorry, I just remembered a previous comment (maybe on Slack? Can't find it now) which noted that our implementation did not match the [paper](https://arxiv.org/abs/1711.05101v3):
![image](https://user-images.githubusercontent.com/4062143/151720940-f4baf4f4-46f4-4ba2-a014-ee41d3fc8581.png).

TL;DR is that (possibly because of an odd choice of variable names in the paper, they use `α` instead of `η` for the learning rate and `η` for something else), the "fixed" version is wrong and the original is correct. Apologies for the noise.

Reply:
oh damn, Brian is right, https://github.com/FluxML/Flux.jl/pull/1612 should be reverted, I got confused by the nomenclature

Reply:
I'll file a PR in Flux